taskcluster/ac_taskgraph/build_config.py

```diff
@@ -31,6 +34,22 @@ def get_path(component):
     return _read_build_config()["projects"][component]["path"]
 
 
+def get_extensions(component):
+    artifact_type = _read_build_config()["projects"][component].get("artifact-type", "aar")
+    if artifact_type not in EXTENSIONS:
+        raise ValueError(
+            "For '{}', 'artifact-type' must be one of {}".format(
+                component, repr(EXTENSIONS.keys()
```

Comment:
Nit: you're missing a `)`, which makes the decision task busted.

Reply:
Oops.  Wasn't sure how to test this locally ;)  We'll see if this works better, and I'll at least try to get it green before you're back :)